src/app/components/FeaturedCourses/featuredcourses.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+import styles from './featuredcourses.module.scss';
+import { Course } from '@/types/course';
+import React, { useState } from 'react';
+import FeatureCourseCard from './featurecoursecard/featurecoursecard';
+
+const featureCourse: Course[] = [
+    {
+        id: 1,
+        category: "Health & Fitness",
+        price: "14.00$",
+        description: "Investing In Stocks The Complete Course! (13 H...",
+        image: "@/public/corleone.svg"
+    },
+    {
+        id: 2,
+        category: "Personal Development",
+        price: "14.00$",
+        description: "Google Analytics Certification - Learn How To...",
+        image: "@/public/courseone.svg"
+    },
+    {
+        id: 3,
+        category: "Productivity",
+        price: "14.00$",
+        description: "Adobe XD for Web Design: Essential Principles",
+        image: "../@/public/courseone.svg"
+    },
+    {
+        id: 4,
+        category: "Music",
+        price: "14.00$",
+        description: "The Python Mega Course: Build 10 Real World ...",
+        image: "C:\Users\samed\Desktop\june-frontend-e-tutor-lms-1\public\coursefour.svg"
+    },
+];
+
+const FeaturedCourses: React.FC = () => {
+    const [courses, setCourses] = useState<Course[]>(featureCourse);
```

Comment:
Burada data değişmeyeceği için state'e ihtiyacımız yok. Yukarıda tanımlanan data direkt map edilebilir